operator/v1/types_machineconfiguration.go

```diff
@@ -56,6 +58,17 @@ type MachineConfigurationSpec struct {
 	// +openshift:enable:FeatureGate=NodeDisruptionPolicy
 	// +optional
 	NodeDisruptionPolicy NodeDisruptionPolicyConfig `json:"nodeDisruptionPolicy"`
+
+
+	// configurationValidationPolicy tells the operator how new machine configurations should be validated.
+	// Valid values are Strict and Relaxed:
+	// Strict: Rejects changes to MachineConfigs if fields that doesn't support to be updated are changed.
```

Comment:
We generally follow this format on OpenShift APIs when talking about what happens when setting a specific allowed value:
```suggestion
	// When set to Strict, changes to MachineConfigs fields that doesn't support to be updated are rejected.
```

What does "doesn't support to be updated" mean? are these the "protected" fields you mention in the description of the Relaxed setting?

Reply:
I've added a bit more information in the new patch, including a link to our docs.
I think the new comment in the new patch may express what I mean a bit better.